environment

```diff
@@ -29,3 +29,11 @@ for i in "${CFU_ROOT}/third_party/python"/* "${CFU_ROOT}/python/"* "${CFU_ROOT}/
 done
 export PYTHONPATH
 
+
+if [ -d "${CFU_ROOT}/third_party/usr/local/bin" ] ; then
+    if [ ! `echo ${PATH} | grep third_party/usr/local/bin` ] ; then
```

Comment:
Maybe it's not a big deal, but this won't work if $PATH contains spaces, because the backticks operator does word splitting on the substituted value. You can replace the backticks with quoted $() operator to avoid word splitting. As in:

    if [ ! "$(echo $PATH | grep ...

My preferred way of writing this kind of condition to check for a substring, is by testing the exit status of `grep -q`, as in:

    if ! grep -q third_party/usr/local/bin <<<$PATH ; then

Reply:
Thanks for pointing this out!  I'll use Alan's suggestion below for improving this.